### DISCUSSION TITLE
MusicBar will damage your computer. You should move it to the trash.

### DISCUSSION THREAD

**Discussion by Kentakoong** [Kentakoong/MusicBar#6] [Announcements]
This issue is occurring because I, the developer of MusicBar haven't enrolled for the Apple Developer Membership yet. 
When I enroll in the program, the issue will be fixed as soon as possible.

My condolences, Kenta.

**Comment by Kentakoong** [comment]
To fix this in the first place. Please clone the repository and build the Application by yourself for now.

**Reply by blaxxius** [reply]
Is there any guide on how to build application by myself? Thanks.

**Reply by Kentakoong** [reply]
I'll do a guide in readme soon! Stay tuned for it.

**Reply by mintyleaf** [reply]
well you can actually do the
`xattr -d com.apple.quarantine MusicBar.app`
and then 
`sudo codesign -f -s - /Applications/MusicBar.app/Contents/MacOS/MusicBar`

get it running, and watch that album art at the center of the track title underneath and get mad because no one in the whole GitHub did a proper menubar extension for music title showing using that native now-playing api
since I'm pretty new at mac'ing, and don't wanna mess with Xcode and other traffic heavy and paid stuff - I can't anything, but really want to =D

**Reply by Kentakoong** [reply]
> Is there any guide on how to build application by myself? Thanks.

Just fInished the Build Documentation. Check it out at BUILD.md

**Comment by Kentakoong** [comment]
Well, this problem is occurring when I have the developer account already...
![Screenshot 2023-04-07 at 22 10 57](https://user-images.githubusercontent.com/35761701/230632365-6dcf15bd-28cc-4fdb-be11-32bc904c616d.png)

**Comment by Kentakoong** [comment]
So, I finally learned how to codesign🤣

It's finally fixed, thank you for waiting!

**Reply by johnmiles27** [reply]
So this app will no longer damage my computer?

**Reply by Kentakoong** [reply]
> So this app will no longer damage my computer?

The pre-released version are now codesigned, so now it would show up as a normal app.
You could check the prerelease version in the releases tab
